.github/workflows/go.yml

```diff
@@ -19,7 +20,7 @@ jobs:
         uses: actions/checkout@v3
 
       - name: Generate mixnet configuration
-        run: cd genconfig && go build && ./genconfig -v -b /conf -o ../docker/voting_mixnet/
+        run: cd genconfig && go build && ./genconfig -v -onlyTransports ${{ matrix.transport }} -b /conf -o ../docker/voting_mixnet/
```

Comment:
does that make it only use one transport protocol for the entire mixnet? e.g. QUIC?

Reply:
it should show if there is a regression/breakage related to a single transport type - because if onlyTransports is not specified all transports are used.